PowerPointLabs/PowerPointLabs/TextCollection.cs

```diff
@@ -18,161 +18,197 @@ public class TextCollection
+            "To perform this action, select the slide(s) with the notes, then click this button.";
         public const string RemoveCaptionsButtonSupertip =
-            "Removes captions added using Captions Lab from the selected slides.";
-        public const string RemoveAllNotesButtonSupertip = "Remove notes from note pane of selected slides.";
+            "Removes captions added using Captions Lab from the selected slides.\n\n" +
+            "To perform this action, select the slide(s) to remove the captions from, then click this button.";
+        public const string RemoveAllNotesButtonSupertip = 
+            "Remove notes from Notes pane of selected slides.\n\n" +
+            "To perform this action, select the slide(s) to remove the notes from, then click this button.";
         # endregion
 
         # region Highlight Lab
         public const string AddSpotlightButtonSupertip =
-            "Creates a spotlight effect for a selected shape.\n\n" +
-            "To activate, draw a shape that the spotlight should outline, select it, then click this button.";
+            "Create a spotlight effect for the slide using selected shapes.\n\n" +
+            "To perform this action, draw a shape that the spotlight should outline, select it, then click this button.";
         public const string SpotlightPropertiesButtonSupertip =
-            "Set the Spotlight properries.";
+            "Configure the settings for Spotlight.";
         public const string HighlightBulletsTextButtonSupertip =
-            "Highlights selected bullet points by changing the text's color.\n\n" +
-            "To activate, select the bullet points to highlight, then click this button.";
+            "Highlight selected bullet points by changing the font color.\n\n" +
+            "To perform this action, select the bullet points to highlight, then click this button.";
         public const string HighlightBulletsBackgroundButtonSupertip =
-            "Highlights selected bullet points by changing the text's background color.\n\n" +
-            "To activate, select the bullet points to highlight, then click this button.";
+            "Highlight selected bullet points by changing the text background color.\n\n" +
+            "To perform this action, select the bullet points to highlight, then click this button.";
         public const string HighlightTextFragmentsButtonSupertip =
-            "Highlights the selected text fragments.\n\n" +
-            "To activate, select the text to highlight, then click this button.";
+            "Highlight the selected text fragments.\n\n" +
+            "To perform this action, select the text fragment(s) to highlight, then click this button.";
         public const string RemoveHighlightButtonSupertip =
-            "Removes all Highlighting from the current slide.";
+            "Removes all Highlighting from the current slide." +

```

Comment:
Needs a whitespace after "current slide.".
Also for Make Transparent, Blur All Except Selected, Recolor Remainder, Recolor All Except Selected, Create Text/Visual/Sidebar Agenda, Remove Agenda, Sync Agenda.